tests/features/support/step-definitions.js

```diff
@@ -58,3 +58,6 @@ defineStep('I wait for (a/an/the)( ){target}', (qa) =>
 
 defineStep('I find/identify/see/spot a link to {string} in (a/an/the)( ){target}', (linkHref, qa) =>
   i.seeALinkInTarget(linkHref, qa));
+
+defineStep('I should have/see/see/spot {int} {target}', (count, qa) =>
```

Comment:
add an optional (s) after {target} for better pluralization.

Then update the usage in the features to be like:
```And I should have 2 `filter badge`s```